app/legacy_lib/query_supporters.rb

```diff
@@ -228,10 +228,14 @@ def self.full_filter_expr(np_id, query)
       date_ago = Timespan::TimeUnits[query[:MAX_payment_before]].utc
       expr = expr.and_where("payments.max_date < timezone(COALESCE(nonprofits.timezone, \'UTC\'), timezone(\'UTC\', $date)) OR payments.count = 0", date: date_ago)
     end
+    multiple_strings = []
+    query[:search].to_s.split(' ').each do |s|
+      multiple_strings << "OR supporters.name ILIKE '%#{s}%'"
```

Comment:
Not a major issue since we're not deploying this version but this is unsafe since it skips the SQL injection protections.